content/ar/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html

```diff
@@ -0,0 +1,101 @@
+---
+title: استخدام مينيكيوب لإنشاء عنقود
+weight: 10
+description: |-
+  تعلم ما هو عنقود كوبيرنيتيس.
+  تعلم ما هو مينيكيوب.
+  أنشئ عنقود كوبيرنيتيس.
+---
+
+<!DOCTYPE html>
+
+<html lang="ar">
+
+<body>
+
+<div class="layout" id="top">
+
+    <main class="content">
+
+        <div class="row">
+
+     <div class="col-md-8">
+        <h3>الأهداف</h3>
+                <ul>
+                    <li>اطّلع على مفهوم عنقود كوبيرنيتيس.</li>
+                    <li>تعرّف على ماهية مينيكيوب.</li>
+                    <li>أنشئ عنقود كوبيرنيتيس على جهاز الكمبيوتر الخاص بك.</li>
+                </ul>
```

Comment:
أفضل أن لا تكون الافعال في نقاط الاهداف على غير صيغة الامر:
```suggestion
                    <li>الاطّلاع على مفهوم عنقود كوبيرنيتيس.</li>
                    <li>التعرّف على ماهية مينيكيوب.</li>
                    <li>إنشاء عنقود كوبيرنيتيس على جهاز الكمبيوتر الخاص بك.</li>
                </ul>
```